fetch_and_process_api_call/as/verify.go

```diff
@@ -0,0 +1,53 @@
+package as
```

Comment:
It looks like you are not using the verification tools in this demo and so this file should be able to be removed

Reply:
removing